components/video-hearings/variables.tf

```diff
@@ -95,4 +95,10 @@ variable "deploy_acme" {
 variable "expiresAfter" {
   description = "Date when Sandbox resources can be deleted. Format: YYYY-MM-DD"
   default     = "3000-01-01"
-}
\ No newline at end of file
+}
+
+# Defined here as we have an input in the tfvar file but this variable is not used within this component
+variable "create_custom_roles" {
```

Comment:
Just a thought, should this default to false and be overridden  in tenants except the prod one? Heads to the goal of moving away from here

Reply:
Fair point

Reply:
False set in module and on enterprise component, true remains on vh component due to the use of prod.tfvars across both production tenants.